andela/travel_tool_front
# #161376711 Modify Room Information

#### What does this PR do?
This PR ensures that the data for room details are correctly displayed in each boxes

#### Description of Task to be completed?
- get data for the number of rooms
- change name from Available rooms to Room capacity

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `ft-modify-room-information-161376711`
- Install dependencies - `yarn install`
- Start the server  - `make start`
- checkout to develop on the server side using this [link](https://github.com/andela/travel_tool_back)
- Navigate to `localhost:3000` in a browser
- Navigate to the Residence page using the link provided in the accommodation section on the left side navigation bar
- You should see different boxes with data showing `Room capacity etc`
#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[#161376711](https://www.pivotaltracker.com/story/show/161376711)

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-10-26 at 7 53 14 am" src="https://user-images.githubusercontent.com/26988297/47549516-3c4b7c80-d8f4-11e8-8892-42ed5627a5f7.png">


#### Questions:
